test/search/metaheuristics/MOSA.test.ts

```diff
@@ -104,7 +104,8 @@ describe("Test MOSA", function () {
 
   it("Test Preference Sorting", () => {
     // This test requires a defined population size.
-    Properties.population_size = 4;
+    CONFIG.populationSize = 4;
+    CONFIG.population_size = 4;
```

Comment:
```suggestion
    CONFIG.populationSize = 4;
```

Why do we set two different population sizes? Probably this is left over from the rename

Reply:
Not exactly, yargs always provides two options: kebabcase, and camelcase.
We only use camelcase though so I could remove it.
What do you think.

Reply:
I would suggest we remove it

Reply:
We can document how we expect the configuration to be used

Reply:
Good idea, removed it.